spec/stashcache.spec

```diff
@@ -9,18 +9,18 @@ BuildArch: noarch
 Source0:   %{name}-%{version}.tar.gz
 
 BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```

Comment:
As long as we're in here, we should remove this line because it was only required for RHEL 5.